src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -418,7 +422,9 @@ private NamespaceDeclarationSyntax ParseNamespaceDeclarationCore()
                 }
 
                 Debug.Assert(initialBadNodes == null); // init bad nodes should have been attached to open brace...
-                return _syntaxFactory.NamespaceDeclaration(namespaceToken, name, openBrace, body.Externs, body.Usings, body.Members, closeBrace, semicolon);
+                return _syntaxFactory.NamespaceDeclaration(
+                    attributeLists, modifiers.ToList(),
```

Comment:
>modifiers.ToList() [](start = 36, length = 18)

nit: I didn't understand why we need to do `.ToList()` on `modifiers`, but not `attributeLists`.

Reply:
I agree.  It's super bizarre (but follows the parser everywhere else).  It has to do with one being a token list, and one being a list of nodes with an actual type.  I'd prefer to not change anything here as it woudl have a large impact on all of LanguageParser.